src/ClassicUO.Client/LegionScripting/docs/API.md

```
@@ -0,0 +1,2744 @@
+
+<details><summary><h3>SetSharedVar(name, value)</h3></summary>
+
+ Set a variable that is shared between scripts.  
+ Example:  
+ ```py  
+ API.SetSharedVar("myVar", 10)  
+ ```  
+  
+
+**Parameters**  
+| Name | Type | Optional | Description |
```

markdownlint-cli2 (0.17.2)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)